dotCMS/src/main/java/com/dotcms/security/apps/AppsAPIImpl.java

```diff
@@ -85,6 +104,35 @@ private String toJsonAsString(final AppSecrets object) throws DotDataException {
         }
     }
 
+    /**
+     * This method takes a byte array and converts its contents into a char array
+     * No String middle man is created.
+     * https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-passwords
+     * @param bytes
+     * @return
+     */
+    char[] bytesToCharArrayUTF(byte[] bytes) throws IOException {
+        final BufferedReader reader = new BufferedReader(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure that resources like this BufferedReader object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)